tests/angular_component_examples/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -76,11 +77,12 @@ class     TestComponent {}
 =====================================output=====================================
 @Component({
   selector: "app-test",
-  template: \`
-    <ul>
-      <li>test</li>
-    </ul>
-  \`,
+  template:
+    \`
```

Comment:
Changes like this definitely don't look good. I'd say the current behavior should be kept for multiline template literals.